core_banking_bootstrap.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+function print_usage() {
+  echo "[INFO]        Usage:"
+  echo "[INFO]              ./core_banking_bootstrap.sh [skipBuild]"
+  echo "[INFO]              skipBuild: optional flag to skip images build"
+  echo "[INFO]        Note: 1) must be run from the project root dir"
+  echo "[INFO]              2) docker must be running"
+}
+
+if [[ -n "$1" && "$1" != "skipBuild" ]]; then
+  echo "[ERROR] Unknown flag $1."
+  print_usage
+  exit 1
+fi
+
+echo "[INFO] Checking if docker daemon is running..."
+rep=$(curl -s --unix-socket /var/run/docker.sock http://ping >/dev/null)
+status=$?
+if [ "$status" == "7" ]; then
+  echo '[ERROR] docker is not running, please start docker.'
+  print_usage
+  exit 1
+fi
+
+echo "[INFO] Checking the current path..."
+path=$(pwd)
+if [[ ${path} != *"core-banking" ]]; then
+  echo "[ERROR] Current path ${path}, please run from the project root dir .../../core-banking."
```

Comment:
How does it know that it is `../../core-banking`? Also if only the ending of the path is checked, it could be run from any random directory that is named `core-banking`. I'd suggest to check instead if current directory contains the script file, or simply check variable `$0`, it should give the command that was used to launch the script e.g. `../../core_banking_bootstrap.sh` should be when it is launched from subdir, `./core_banking_bootstrap.sh` when launcher from root.